src/book/02-projects/01-nubots/03-build-system.mdx

```diff
@@ -183,7 +183,7 @@ This tool uses NBS files recorded from the robot(s) to gather train and test dat
 
 ##### Format
 
-NUbots uses three different code formatters. `clang-format` is used to format C++ and protobuf files, `black` and `isort` are used to format python files, and `cmake-format` is used to format CMake files.
```

Comment:
Maybe just change this to "NUbots uses several different code formatters" That way we're not wrong when we add more in the future